testing/src/cypress.env.json.sample

```diff
@@ -17,6 +17,7 @@
     "MINISTRY_LOGIN": "/aest/login",
     "STUDENT_LOGIN": "/student/login",
     "CREATE_INSTITUTION_LOCATION": "/api/institutions/location",
-    "INSTITUTIONS_API": "/api/institutions"
+    "INSTITUTIONS_API": "/api/institutions",
```

Comment:
Just a heads up, the endpoints are not environment dependents. They will be constant across all the environments and can be defined as constants for the tests.

Reply:
Yeah, sure. we will handle that in the PR handling ENV changes changes